demo-shell/src/app/app.routes.ts

```diff
@@ -42,37 +42,58 @@ import { ProcessDetailsCloudDemoComponent } from './components/cloud/process-det
 import { FormCloudDemoComponent } from './components/app-layout/cloud/form-demo/cloud-form-demo.component';
 import { DemoErrorComponent } from './components/error/demo-error.component';
 import { ProcessCloudLayoutComponent } from './components/cloud/process-cloud-layout.component';
-import { SearchFilterChipsComponent } from './components/search/search-filter-chips.component';
+import { AppSearchFilterChipsComponent } from './components/search/search-filter-chips.component';
+import { FileViewComponent } from './components/file-view/file-view.component';
+import { SettingsComponent } from './components/settings/settings.component';
+import { AppLoginComponent } from './components/login/login.component';
+import { TaskListDemoComponent } from './components/task-list-demo/task-list-demo.component';
+import { ProcessListDemoComponent } from './components/process-list-demo/process-list-demo.component';
+import { AppCardViewComponent } from './components/card-view/card-view.component';
 
 export const appRoutes: Routes = [
-    { path: 'login', loadChildren: () => import('./components/login/login.module').then(m => m.AppLoginModule) },
+    { path: 'login', component: AppLoginComponent },
```

Comment:
AppLoginComponent is standalone, so it can be lazy loaded like
```suggestion
    { path: 'login', loadComponent: () => import('./components/login/login.component').then(c => c.AppLoginComponent },
```

The same goes for all the further components in the appRoutes